src/studio/src/designer/frontend/shared/components/molecules/AltinnModal.tsx

```diff
@@ -98,7 +98,7 @@ export class AltinnModal extends React.Component<IAltinnModalComponentProvidedPr
           open={this.props.isOpen}
           className={this.props.classes.scroll}
           hideBackdrop={this.props.hideBackdrop}
-          onBackdropClick={this.props.allowCloseOnBackdropClick === false ? null : this.props.onClose}
+          onClose={this.props.allowCloseOnBackdropClick === false ? null : this.props.onClose}
```

Comment:
will it no longer be possible to click on the backdrop to close modals after this?

Reply:
It will,  `onBackdropClick` is deprecated for `onClose`